pkg/apis/mongodb/v1/mongodb_types.go

```diff
@@ -31,6 +37,9 @@ type MongoDBSpec struct {
 
 // MongoDBStatus defines the observed state of MongoDB
 type MongoDBStatus struct {
+	MongoURI string `json:"mongoUri"`
+	Members  int    `json:"members"`
```

Comment:
If this field is going to be a copy of what we have in `spec`, then it should be removed.